packages/components/switch/src/SwitchInput.tsx

```diff
@@ -82,6 +95,13 @@ export const Input = React.forwardRef<HTMLButtonElement, InputProps>(
         checked={checked}
         defaultChecked={defaultChecked}
         onCheckedChange={handleCheckedChange}
+        id={id || controlledId}
+        name={name || controlledName}
+        required={required || isRequired}
+        aria-required={required || isRequired}
```

Comment:
There is not need of doing this. The spread operator will overwrite `{...rest}` these props

Reply:
I'm not sure to understand :thinking: Even if the `required` attribute is correctly implemented with corresponding aria attribute, the `isRequired` fallback wouldn't be in there, don't you think? Or am I missing something?